engine/buildkit/filesync.go

```diff
@@ -226,7 +227,8 @@ func (c *Client) LocalDirExport(
 	}
 
 	ctx = engine.LocalExportOpts{
-		Path: destPath,
```

Comment:
Oh, so this defaulted to 'delete everything' before? I've had files mysteriously disappear recently, maybe this expains it.

Reply:
No the behavior before was to not delete any files, that's unchanged here. So something else must have been afoot 🔎